zebrad/tests/acceptance.rs

```diff
@@ -40,13 +41,15 @@
 //!
 //! $ cargo test sync_large_checkpoints_mempool_mainnet -- --ignored --nocapture
 //!
-//! $ sudo mkdir /zebrad-cache
-//! $ sudo chmod 777 /zebrad-cache
+//! $ export ZEBRA_CACHED_STATE_DIR="/zebrad-cache"
+//! $ sudo mkdir -p "$ZEBRA_CACHED_STATE_DIR"
+//! $ sudo chmod 777 "$ZEBRA_CACHED_STATE_DIR"
 //! $ export FULL_SYNC_MAINNET_TIMEOUT_MINUTES=600
 //! $ cargo test full_sync_mainnet -- --ignored --nocapture
```

Comment:
Can you provide me a link of the CI running this pull request and this test successfully ?

Reply:
This would be the relevant outputs:
- https://github.com/ZcashFoundation/zebra/actions/runs/10884701795/job/30342588717?pr=8865#step:6:31
- https://github.com/ZcashFoundation/zebra/actions/runs/10884701795/job/30342595236?pr=8865#step:6:38